apps/frontend/src/renderer/components/github-prs/components/PRLogs.tsx

```
@@ -235,20 +316,205 @@ function PhaseLogSection({ phase, phaseLog, isExpanded, onToggle, isStreaming =
+  const summary = summaryParts.join(', ') || `${entries.length} operations`;
+
+  return (
+    <div className="rounded-md border border-border/50 bg-secondary/10 overflow-hidden">
+      <button
+        onClick={onToggle}
+        className={cn(
+          'w-full flex items-center justify-between p-2 transition-colors',
+          'hover:bg-secondary/30',
+          isExpanded && 'bg-secondary/20'
+        )}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)